File: entity/sku.go
package entity

import "time"

type Sku struct {
	SkuID     int64     `gorm:"primaryKey;column:sku_id;type:bigint;not null" json:"-"`
	SpuID     int64     `gorm:"column:spu_id;type:bigint" json:"spuId"`
	SkuCode   string    `gorm:"column:sku_code;type:varchar(16)" json:"skuCode"`
	SkuDesc   string    `gorm:"column:sku_desc;type:varchar(32)" json:"skuDesc"`
	CreatedAt time.Time `gorm:"column:created_at;type:datetime" json:"createdAt"`
	UpdatedAt time.Time `gorm:"column:updated_at;type:datetime" json:"updatedAt"`
}

// TableName get sql table name.获取数据库表名
func (m *Sku) TableName() string {
	return "sku"
}

File: fontend/MyAntDesignApp2/Models/Sku.cs
using System.ComponentModel;

namespace WmsApp.Models
{
    public class Sku : CommonDate
    {
        public long SpuID { get; set; }
        public long SkuID { get; set; }
        [DisplayName("编码")]
        public string SkuCode { get; set; }
        [DisplayName("描述")]
        public string SkuDesc { get; set; }
    }

}
